src/WinGet.RestSource.IntegrationTest/Tests/Functions/PackageManifestFunctionsTests.cs

```diff
@@ -206,7 +220,7 @@ public async Task GetPackages()
         [Fact]
         public async Task SearchUsingQuery()
```

Comment:
>SearchUsingQuery [](http://example.com/codeflow?start=26&length=16)

Separate Tests

Reply:
This has been addressed in manifest search tests, code refactor coming in the next pull request.